README.md

```diff
@@ -3,7 +3,7 @@
 - LinkFree connects audiences to all of your content with just one link. It is an open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript.
 - It was initially created on a YouTube [live stream](https://www.youtube.com/watch?v=Jorl_vcp-Ew).
 
-![Screenshot](https://user-images.githubusercontent.com/91655303/145024633-559c1a21-240b-4900-9f06-7917a6f0d241.png)
+![Eddie Jaoude's LinkFree profile GIF](https://user-images.githubusercontent.com/82458069/145549448-b1feb5df-78aa-4286-a709-6985f0dade4c.gif)
```

Comment:
I think it will be better if we just say "Eddie Jaoude's LinkFree profile". No need to include GIF here.

Reply:
In my view, it's good to add a ***GIF*** label for better understandability, readability, and differentiating between GIF and Image.